drivers/misc/nordic_vpr_launcher/nordic_vpr_launcher.c

```diff
@@ -63,16 +68,20 @@ static int nordic_vpr_launcher_init(const struct device *dev)
 	(DT_REG_ADDR(node_id) +                                                                    \
 	 COND_CODE_0(DT_FIXED_PARTITION_EXISTS(node_id), (0), (DT_REG_ADDR(DT_GPARENT(node_id)))))
 
+#define NEEDS_COPYING(inst) UTIL_AND(DT_INST_NODE_HAS_PROP(inst, execution_memory),                \
```

Comment:
won't Kconfig dependency prevent this to ever be true?

Reply:
Why? If no VPR node has the `execution-memory` property, the driver will not be activated at all. But if any has it, this macro will evaluate to 1 for those that have both `execution-memory` and `source-memory` properties.

Reply:
right, we can have > 1 entry